src/devices/tuya.ts

```diff
@@ -1136,7 +1137,14 @@ const definitions: Definition[] = [
             await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
             await reporting.batteryPercentageRemaining(endpoint);
         },
-        exposes: [e.water_leak(), e.tamper(), e.battery_low(), e.battery()],
+        exposes: (device, options) => {
+            const exps: Expose[] = [e.water_leak(), e.battery_low(), e.battery()];
+            if (!device || device.manufacturerName === '_TZ3000_mugyhz0q') {
```

Comment:
```suggestion
            if (!device || device.manufacturerName !== '_TZ3000_mugyhz0q') {
```

I guess you don't want to add tamper in case of `_TZ3000_mugyhz0q`?

Reply:
Thanks, that was exactly the idea. I love that's a possibility.